src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -274,7 +272,20 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
       if (local_config_GM_kappa_lat_depth_variable) then
         !$omp do schedule(runtime)
```

Comment:
needs private (k, maxN, BruntVaisalaFreqTopEdge)

Reply:
thanks I will add this